.github/dependabot.yml

```diff
@@ -0,0 +1,47 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "npm" # See documentation for possible values
```

Comment:
@anthonyroux 

1. `package-ecosystem` should be `yarn` because that's what we are using.
2. I don't think we need a rule for each example, because we are using yarn workspaces & we only have a single `yarn.lock` file. So we only need the root one.

Reply:
This monorepo only uses `yarn`, if anyone uses `npm` that's a mistake. Also, we shouldn't have any lockfiles other than the root `yarn.lock` if we commit any, then that's also a mistake.